errors/code.go

```diff
@@ -13,6 +13,12 @@ const (
 	ErrorCodeEmpty = Code("")
 )
 
+// GetCode returns the error code. If the error doesn't contains
+// an error code, returns ErrorCodeEmpty
+func GetCode(err error) Code {
```

Comment:
If this function is going to be the default, due to naming convention, then the implementation should be moved here, and `GetErrorCode` should point to this function.

Reply:
Changed by amend